src/ngx_http_lua_ssl_certby.c

```diff
@@ -880,6 +880,36 @@ ngx_http_lua_ffi_ssl_server_name(ngx_http_request_t *r, char **name,
 #endif
 }
 
```

Comment:
Style: nginx requires 2 blank lines to separate adjacent C function definitions.

Reply:
@smartwjw Seen this comment?